src/main/scala/nl.knaw.dans.easy.bagstore/component/BagProcessingComponent.scala

```diff
@@ -159,9 +158,7 @@ trait BagProcessingComponent extends DebugEnhancedLogging {
         Files.createDirectory(extractDir)
         val zip = extractDir.resolve("bag.zip")
         FileUtils.copyInputStreamToFile(is, zip.toFile)
-        new ZipFile(zip.toFile) {
-          setFileNameCharset(StandardCharsets.UTF_8.name)
```

Comment:
is this no longer necessary? or should we actually use the new `setCharset` instead?

Reply:
No longer necessary! The latest version of Zip4j has `UTF8` as its default charset. (see https://github.com/srikanth-lingala/zip4j/blob/master/src/main/java/net/lingala/zip4j/ZipFile.java#L92)